api/src/main/java/org/openmrs/api/db/SerializedObject.java

```diff
@@ -9,22 +9,45 @@
  */
 package org.openmrs.api.db;
 
+import org.hibernate.annotations.GenericGenerator;
+import org.hibernate.annotations.Parameter;
 import org.openmrs.BaseChangeableOpenmrsMetadata;
 import org.openmrs.serialization.OpenmrsSerializer;
 
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.Table;
+
 /**
  * Object representation of a Serialized Object as stored in the database.
  */
+@Entity
+@Table(name = "serialized_object")
 public class SerializedObject extends BaseChangeableOpenmrsMetadata {
-	
+
+	@Id
+	@GeneratedValue(strategy = GenerationType.SEQUENCE, generator = "serialized_object_id_seq")
```

Comment:
Is there any reason why you have not used `GenerationType.IDENTITY`?

Reply:
@dkayiwa  The earlier implementation utilized GenerationType.SEQUENCE, and this choice is also reflected in [this pull request](https://github.com/openmrs/openmrs-core/pull/4388/files#72), which I used as a reference. Some of the code reviews suggested a modification from GenerationType.IDENTITY to GenerationType.SEQUENCE. 

 Please let me know, if I need to change `sequence` to `Identity`.

Reply:
@dkayiwa  any update on this?